examples/xgboost/job_config_gen.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+python3 utils/prepare_job_config.py --site_num 5 --train_mode bagging --split_method exponential --lr_mode scaled
+python3 utils/prepare_job_config.py --site_num 5 --train_mode bagging --split_method exponential --lr_mode uniform
+python3 utils/prepare_job_config.py --site_num 5 --train_mode bagging --split_method uniform --lr_mode uniform
+python3 utils/prepare_job_config.py --site_num 5 --train_mode cyclic --split_method exponential --lr_mode uniform
+python3 utils/prepare_job_config.py --site_num 5 --train_mode cyclic --split_method uniform --lr_mode uniform
+
+python3 utils/prepare_job_config.py --site_num 20 --train_mode bagging --split_method square --lr_mode scaled
+python3 utils/prepare_job_config.py --site_num 20 --train_mode bagging --split_method square --lr_mode uniform
+python3 utils/prepare_job_config.py --site_num 20 --train_mode bagging --split_method uniform --lr_mode uniform
+python3 utils/prepare_job_config.py --site_num 20 --train_mode cyclic --split_method square --lr_mode uniform
```

Comment:
why site 20 use square VS site 5 use exponential

Reply:
consider the data size and non-uniform-ness needed to produce performance drop, added to readme